testutil/mock/privval.go

```diff
@@ -0,0 +1,50 @@
+package mock
+
+import (
+	"github.com/tendermint/tendermint/crypto"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	tmtypes "github.com/tendermint/tendermint/types"
+
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+)
+
+var _ tmtypes.PrivValidator = PV{}
```

Comment:
Why do we need to implement this type? Was it not needed before?

Reply:
bump @aleem1314

Reply:
Simapp didn't use validators (it used an empty validator set)

This file was taken from ibc-go. We added it in ibc-go for [this reason](https://github.com/cosmos/cosmos-sdk/issues/7177). In ibc-go we need the priv validator for signing headers (not sure if that is used in the SDK)?